src/signals/incident-management/containers/IncidentSplitContainer/components/SplitDetail/index.js

```diff
@@ -1,43 +1,63 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import { Heading } from '@datapunt/asc-ui';
+import styled, { ascDefaultTheme } from '@datapunt/asc-core';
 import { string2date, string2time } from 'shared/services/string-parser/string-parser';
 import { getListValueByKey } from 'shared/services/list-helper/list-helper';
 
-import './style.scss';
+const StyledWrapper = styled.aside`
+  dt {
+    color: ${ascDefaultTheme.colors.tint.level5};
+    margin-bottom: 5px;
+  }
+
+  dd {
+    font-family: 'Avenir Next LT W01 Demi', arial, sans-serif;
+    margin-bottom: 16px;
+  }
+`;
+
+const StyledH4 = styled(Heading)`
+  font-weight: normal;
```

Comment:
Instead of doing
```jsx
<StyledH4
  data-testid="splitDetailTitle"
  $as="h4"
>
```
you could also do this:
```jsx
const StyledH4 = styled(Heading).attrs({
   $as': 'h4',
})`
  font-weight: normal;
  margin-bottom: 8px;
`;

<StyledH4 data-testid="splitDetailTitle">
```
The result would be the same; it's just a matter of preference.

Reply:
gedaan